### PR TITLE
Add sections Getting Help and Opening Issues

### PR DESCRIPTION
**Pull request by Dominitio**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,9 +26,10 @@ Please see the link below for more detail on TypeScript 3.8.x:
 * [TypeScript 3.8.x](https://www.npmjs.com/package/typescript)
```
Comment (byronlin13) [inline README.md:26]: On L8

makes use of Amazon QLDB.

Comment (byronlin13) [inline README.md:26]: Should we specify min Node.JS version here?

File: README.md [hunk 2]
```diff
@@ -26,9 +26,10 @@ Please see the link below for more detail on TypeScript 3.8.x:
...
 
 
-
 ## Getting Started
 
+Please see the [Quickstart guide for the Amazon QLDB Driver for Node.js](https://docs.aws.amazon.com/qldb/latest/developerguide/driver-quickstart-nodejs.html).
+
 To use the driver, in your package that wishes to use the driver, run the following:
```
Comment (byronlin13) [inline README.md:33]: This section isn't in all the drivers, how are we going to unify this?

Comment (Dominitio) [reply README.md:33]: We should add to the others that don't have it.

File: README.md (hunk 1 above)
Comment (Dominitio) [reply README.md:26]: Done

Comment (Dominitio) [reply README.md:26]: We'll have to find out the min Node.js version.

File: README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@
 [![license](https://img.shields.io/badge/license-Apache%202.0-blue)](https://github.com/awslabs/amazon-qldb-driver-nodejs/blob/master/LICENSE)
 [![AWS Provider](https://img.shields.io/badge/provider-AWS-orange?logo=amazon-aws&color=ff9900)](https://aws.amazon.com/qldb/)
 
-This is the Node.js driver for Amazon Quantum Ledger Database (QLDB), which allows Node.js developers to write software that makes use of AmazonQLDB.
+This is the Node.js driver for Amazon Quantum Ledger Database (QLDB), which allows Node.js developers to write software that makes use of Amazon QLDB.
```
Comment (kylepbit) [inline README.md:9]: Go has "For getting started with the driver, see Go and Amazon QLDB." and a link to the getting started, which seems like it belongs here.

File: README.md [hunk 4]
```diff
@@ -26,9 +26,10 @@ Please see the link below for more detail on TypeScript 3.8.x:
...
 
 
-
 ## Getting Started
 
+Please see the [Quickstart guide for the Amazon QLDB Driver for Node.js](https://docs.aws.amazon.com/qldb/latest/developerguide/driver-quickstart-nodejs.html).
+
 To use the driver, in your package that wishes to use the driver, run the following:
 
 ```npm install amazon-qldb-driver-nodejs```
```
Comment (kylepbit) [inline README.md:35]: The See Also section is missing items vs. the Python version?

File: README.md [hunk 5]
```diff
@@ -106,6 +107,19 @@ TypeDoc is used for documentation. You can generate HTML locally with the follow
 
 ```npm run doc```
 
+## Getting Help
+
+Please use these community resources for getting help.
+* Ask a question on StackOverflow and tag it with the [amazon-qldb](https://stackoverflow.com/questions/tagged/amazon-qldb) tag.
+* Open a support ticket with [AWS Support](http://docs.aws.amazon.com/awssupport/latest/user/getting-started.html).
```
Comment (kylepbit) [inline README.md:114]: Missing the entry for the QLDB Forum that's in the Python version.

**Review by kylepbit — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Dominitio) [reply README.md:9]: Done

File: README.md (hunk 4 above)
Comment (Dominitio) [reply README.md:35]: Added entries in See Also section

File: README.md (hunk 5 above)
Comment (Dominitio) [reply README.md:114]: Added

File: README.md [hunk 6]
```diff
@@ -5,7 +5,9 @@
 [![license](https://img.shields.io/badge/license-Apache%202.0-blue)](https://github.com/awslabs/amazon-qldb-driver-nodejs/blob/master/LICENSE)
 [![AWS Provider](https://img.shields.io/badge/provider-AWS-orange?logo=amazon-aws&color=ff9900)](https://aws.amazon.com/qldb/)
 
-This is the Node.js driver for Amazon Quantum Ledger Database (QLDB), which allows Node.js developers to write software that makes use of AmazonQLDB.
+This is the Node.js driver for Amazon Quantum Ledger Database (QLDB), which allows Node.js developers to write software that makes use of Amazon QLDB.
```
Comment (kylepbit) [inline README.md:8]: The 'Amazon Quantum Ledger Database (QLDB)` should be a link as it is on other pages.

**Pull request merged**